src/lib/structures/Command.ts

```diff
@@ -160,6 +161,7 @@ export class Command {
 				this.run = options.run || Command.defaults?.run;
 				this.onError = options.onError || Command.defaults?.onError;
 				this.autoDefer = options.autoDefer || Command.defaults?.autoDefer;
+				this.options = options;
```

Comment:
Put this before zod validation

https://github.com/SebestikCZ/GCommands/blob/c890545d3ce22de37df900aaa2f86db730e89455/src/lib/structures/Command.ts#L135-L136